x-pack/test/siem_cypress/config.ts

```diff
@@ -34,7 +34,7 @@ export default async function({ readConfigFile }: FtrConfigProviderContext) {
       serverArgs: [
         ...xpackFunctionalTestsConfig.get('esTestCluster.serverArgs'),
         // define custom es server here
-        'xpack.security.authc.api_key.enabled=true',
+        // API Keys is enabled at the top level
```

Comment:
👍 let us know if this doesn't work.

@MadameSheema, FYI for here.

Reply:
@MadameSheema @FrankHassanabad This fixed our issue. The new test failure is unrelated. Sweet!